.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**Motivation**:
```

Comment:
I think that this is better to use header, instead of bold.

Reply:
Which header should you recommend?

Reply:
I don't care about the header level, but I'm using the second header `##` in the company.

Reply:
I use `###` 😄